### PR TITLE
Update EKS deployment

### PR DESCRIPTION
**Pull request by orouz**
### Summary of your changes

this PR 
- updates the readme file for deploying an `eks` cluster with additional instructions on how to connect deployed cluster to a running EC deployment. 
- sets the k8s version for the `simple-cluster.yml` config as `1.23`

### CODE REVIEW

File: deploy/eks/Readme.md
Comment (orouz) [inline deploy/eks/Readme.md:1]: mostly markdown formatting changes

File: deploy/eks/Readme.md [hunk 1]
```diff
@@ -61,10 +63,16 @@ See [references](#useful-references) section below for links to installation doc
    eksctl delete cluster --region=eu-west-1 --name=<your_cluster_name> --wait
    ```
 
+### Connect cluster to an EC deployment
+
+1. `aws eks update-kubeconfig --region <region> --name <name_of_cluster>`
+2. verify current k8s is `eksctl` (`kubectl config current-context` should include `eksctl` )
+3. install KSPM EKS in kibana. provide your own credentials (`cat ~/.aws/credentials`)
+4. `kubectl apply -f <K8S_YAML_FROM_FLYOUT>
```
Comment (gurevichdmitry) [inline deploy/eks/Readme.md:71]:
```suggestion
4. `kubectl apply -f <K8S_YAML_FROM_FLYOUT>`
```

File: deploy/eks/Readme.md [hunk 2]
```diff
@@ -61,10 +63,16 @@ See [references](#useful-references) section below for links to installation doc
...
 
 ## Useful references
 
-- [ekcstl config file examples](https://github.com/weaveworks/eksctl/tree/main/examples)  for different deployment scenerios
+- [ekcstl config file examples](https://github.com/weaveworks/eksctl/tree/main/examples) for different deployment scenerios
```
Comment (gurevichdmitry) [inline deploy/eks/Readme.md:75]:
```suggestion
- [ekcstl config file examples](https://github.com/weaveworks/eksctl/tree/main/examples) for different deployment scenarios
```

File: deploy/eks/Readme.md [hunk 3]
```diff
@@ -47,12 +49,12 @@ See [references](#useful-references) section below for links to installation doc
...
 
 2. Run The following command to deploy cluster:
-    ```bash
-    eksctl create cluster --config-file deploy/eks/simple-cluster.yml
-    ```
-    Creation should take around 20 minutes for a simple config.
-    eksctl will create your cluster and automatically add the context to your `~/.kube/config`
-    The you can proceed with as usual with your tool of choice (kubectl/k9s/lens)
+   ```bash
+   eksctl create cluster --config-file deploy/eks/simple-cluster.yml --version 1.23.12
```
Comment (orouz) [inline deploy/eks/Readme.md:53]: create cluster with a supported k8s version

File: deploy/eks/simple-cluster.yml [hunk 4]
```diff
@@ -5,10 +5,11 @@ kind: ClusterConfig
 metadata:
   name:
   region: eu-west-1
+  version: "1.23"
 
 vpc:
-    # Don't change vpc settings
-  id: 	"vpc-6cb55a15"  # (optional, must match VPC ID used for each subnet below)
```
Comment (orouz) [inline deploy/eks/simple-cluster.yml:11]: auto format changes

File: deploy/eks/simple-cluster.yml [hunk 5]
```diff
@@ -5,10 +5,11 @@ kind: ClusterConfig
 metadata:
   name:
   region: eu-west-1
+  version: "1.23"
```
Comment (orouz) [inline deploy/eks/simple-cluster.yml:8]: this is the version we support, required to ensure actual findings matches our current expected findings

**Review by oren-zohar — APPROVED** [review]

**Pull request merged**